alert_rule_tests/README.md

```diff
@@ -0,0 +1,13 @@
+# How to test and run prometheus rules
+
+## Download and install yq tool
+
+    https://kislyuk.github.io/yq/
+
+## Extract rules from k8s asset:
+    cat assets/vsphere_problem_detector/12_prometheusrules.yaml|yq -Y '.spec' > alert_rule_tests/12_prometheusrules.yaml
```

Comment:
I found multiple references to `yq` in openshift/release repo, it may be available during `make test`. Therefore we may not need to store `alert_rule_tests/12_prometheusrules.yaml` in git.

Reply:
Not in the default image at least:

```
cat assets/vsphere_problem_detector/12_prometheusrules.yaml|yq -Y '.spec' > alert_rule_tests/12_prometheusrules.yaml
/bin/sh: yq: command not found
```

I also think we will have to pull ins special image for making `promtool` available..